src/menu-helpers.js

```diff
@@ -53,14 +54,16 @@ function unmerge(menu, item) {
   }
 }
 
-function findMatchingItemIndex(menu, { type, label, submenu }) {
+// function findMatchingItemIndex(menu, { type, label, submenu }) {
```

Comment:
```suggestion
```

Reply:
I see. I delete the line, according to your advice